core/src/main/java/org/apache/iceberg/BaseOverwriteFiles.java

```diff
@@ -95,11 +101,18 @@ public OverwriteFiles caseSensitive(boolean isCaseSensitive) {
   @Override
   public OverwriteFiles validateNoConflictingAppends(Expression newConflictDetectionFilter) {
     Preconditions.checkArgument(newConflictDetectionFilter != null, "Conflict detection filter cannot be null");
```

Comment:
Nit: Should this be `Append conflict detection filter cannot be null` now that we have both appendConflictDetectionFilter and delteConflictDectionFilter?

Reply:
I'll update if it fits on the same line.